src/main/java/kdt/springbootboardjpa/controller/request/SavePostRequest.java

```diff
@@ -0,0 +1,20 @@
+package kdt.springbootboardjpa.controller.request;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@Getter
+public class SavePostRequest {
```

Comment:
record 타입을 많이 사용하셨던 것으로 기억하는 record 타입이 아닌 class로 Request 객체를 생성하신 이유가 있으신가요? 🤔

Reply:
Builder를 사용하고 싶은데, record는 builder가 적용되지 않더라구요. 그래서 이번에는 class로 만들었습니다. 
혹시 용철님은 어떤 방식을 더 선호하시나요?

Reply:
저는 가능하면 record 타입으로 많이 사용하기는 합니다. 해당 부분은 인텔리제이 버그로 보입니당.. 

저는 실행은 잘되네용 🤔 

https://stackoverflow.com/questions/69825016/lomboks-builder-not-detecting-fields-of-the-java-record

```suggestion
public record SavePostRequest(
    String title,
    String content,
    Long createdBy
) {

    @Builder
    public SavePostRequest {
    }
}
```

위와 같이 사용하면 인텔리제이에서도 자동완성 기능 및 에러가 발생하지 않네요..! 😄

Reply:
와우 바로 적용했습니다! 감사합니다